R/tiltIndicatorAfter-package.R

```diff
@@ -13,6 +13,7 @@
 #' @importFrom dplyr join_by
 #' @importFrom dplyr left_join
 #' @importFrom dplyr mutate
+#' @importFrom dplyr mutate_all
```

Comment:
Are we still using `mutate_all()` anywhere? If not we should remove it here and regenerate NAMESPACE via `devtools::document()`.

Reply:
resolved